R/comparison.R

```diff
@@ -0,0 +1,117 @@
+rcmdcheck_comparison <- function(old, new) {
+  stopifnot(is.list(old))
+  stopifnot(inherits(new, "rcmdcheck"))
+
+  # Generate single dataframe of comparisons
+  old_df <- do.call(rbind, lapply(old, checks_as_df))
```

Comment:
Would be good to keep the names of `old`, because that's the CRAN platform for CRAN checks. It should be probably more explicit in the `cran_check_results` results.